lablib/lib/utils.py

```diff
@@ -14,19 +14,73 @@
 log.setLevel(logging.DEBUG)
 
 
-def __call_cmd(cmd: List[str], timeout=0, retries=0) -> Optional[str]:
+def get_vendored_env() -> None:
```

Comment:
I wonder if we shouldn't do this similarly like it is done in AYON, 

- get_ffmpeg_path
- get_oiiotool_path
- get_ocio_path

This way we could add the path explicitly into the subprocess args.

Reply:
here is an example https://github.com/ynput/ayon-core/blob/2582a086248d7618c6a795be138d204a5864b08f/client/ayon_core/lib/vendor_bin_utils.py#L402